index.Rmd

```diff
@@ -284,13 +288,13 @@ In order to achieve these objectives, you should try to follow a few good practi
     - Indentation (like in Python: use spaces or tab to indent code in functions or loops etc.).
     - Consistent use of the assignment operator: either `<-` or `=` in all your code. The former is used by core R and allows assigning in function calls, the latter is shorter and consistent with most other programming languages.
     - Consistent placement of curly braces.
-* [Make your own packages](#__optional__Writing_packages_).
-* Keep a similar [directory structure](../RProjectManagement/index.html) across your projects.
-* Use [version control](../RProjectManagement/index.html) to develop/maintain your projects and packages.
+* [Make your own packages](#__optional__Writing_packages_). 
```

Comment:
The name of the section may have changed. You can copy the URL from the table of contents at the top right. However, at the moment the file has not been reknitted yet, in the reknitted version it might change again.